examples/ola-rdm.cpp

```diff
@@ -212,14 +213,18 @@ void DisplayHelpAndExit(const options &opts) {
  * Dump the list of known pids
  */
 void DisplayPIDsAndExit(uint16_t manufacturer_id,
-                        const PidStoreHelper &pid_helper) {
-  vector<string> pid_names;
-  pid_helper.SupportedPids(manufacturer_id, &pid_names);
-  sort(pid_names.begin(), pid_names.end());
-
-  vector<string>::const_iterator iter = pid_names.begin();
-  for (; iter != pid_names.end(); ++iter) {
-    cout << *iter << endl;
+                        const PidStoreHelper &pid_helper,
+                        bool set_mode) {
+  vector<const PidDescriptor *> pids;
+  pid_helper.SupportedPids(manufacturer_id, &pids);
+  // Remove PIDs that don't support the proper command class (GET with
+  // ola_rdm_get, SET with ola_rdm_set)
+  vector<const PidDescriptor *>::const_iterator it_pids = pids.begin();
+  for (; it_pids != pids.end(); ++it_pids) {
+    if ((set_mode && (*it_pids)->SetRequest() != NULL)
+        || (!set_mode && (*it_pids)->GetRequest() != NULL)) {
```

Comment:
We generally prefer the || at the end of the line:
```suggestion
    if ((set_mode && (*it_pids)->SetRequest() != NULL) ||
        (!set_mode && (*it_pids)->GetRequest() != NULL)) {
```